src/main/java/core/basesyntax/dao/FruitsDaoImpl.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.FruitTransaction;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitsDaoImpl implements FruitsDao {
+    private static final int FIRST_ELEMENT = 0;
+    private static final int SECOND_ELEMENT = 1;
+    private static final int THIRD_ELEMENT = 2;
+    private List<FruitTransaction> fruitsData;

```

Comment:
It's better to initialize 'fruitsData' as a local variable inside the 'getFruitsData' method, since it is only used there.